kubernetes/overlays/prod-dc2-test-med-1/ingestd.yml

```
@@ -0,0 +1,10 @@
+brokers: 134.79.23.221:9094
```

The IP address changed.
```suggestion
brokers: 134.79.23.189:9094
```